user_management/doc/User_management.md

```diff
@@ -134,4 +138,45 @@ will return a boolean
 > 
 > NB : An empty username is considered as not taken
 
-</details>
\ No newline at end of file
+</details>
+
+## `refreshJWT`
+
+### Trade a valid refresh token for an access token
+
+will return an access token when successful
+
+<details>
+ <summary><code>POST</code><code><b>/user/signup/</b></code></summary>
```

Comment:
Wrong endpoint

Reply:
I think i also forgot to update the main documentation